public/app/features/alerting/unified/Templates.tsx

```diff
@@ -1,31 +1,23 @@
-import { Route, Switch } from 'react-router-dom';
+import { Routes, Route } from 'react-router-dom-v5-compat';
 
 import { withErrorBoundary } from '@grafana/ui';
-import { SafeDynamicImport } from 'app/core/components/DynamicImports/SafeDynamicImport';
 
 import { AlertmanagerPageWrapper } from './components/AlertingPageWrapper';
-
-const EditMessageTemplate = SafeDynamicImport(() => import('./components/contact-points/EditMessageTemplate'));
-const NewMessageTemplate = SafeDynamicImport(() => import('./components/contact-points/NewMessageTemplate'));
-const DuplicateMessageTemplate = SafeDynamicImport(
-  () => import('./components/contact-points/DuplicateMessageTemplate')
-);
+import DuplicateMessageTemplate from './components/contact-points/DuplicateMessageTemplate';
+import EditMessageTemplate from './components/contact-points/EditMessageTemplate';
+import NewMessageTemplate from './components/contact-points/NewMessageTemplate';
```

Comment:
This component is already lazy loaded in the [routes file](https://github.com/grafana/grafana/blob/main/public/app/features/alerting/routes.tsx#L111-L119), so it's not necessary to do so here, also it's harder to do with the new router API.